### ISSUE TITLE
Add static factory method to generate random beans with default configuration parameters

### ISSUE THREAD

**Issue by benas** [benas/random-beans#116]
As [requested](https://github.com/benas/random-beans/issues/113#issuecomment-210447511) in #113 , the idea is to add a static method in the `EnhancedRandom` API which allows the user to get a random instance of a given type without the need to create an instance of `EnhancedRandom` with the builder:

```java
public abstract class EnhancedRandom extends Random {

    // existing methods

    /**
     * Generate a random instance of the given type.
     *
     * @param type the target class type
     * @param <T>  the target type
     * @return a random instance of the given type
     */
    public static <T> T random(final Class<T> type) {
        return aNewEnhancedRandomBuilder().build().nextObject(type);
    }

}
```

With this new method, the user can simply write the following (with the static import) when default parameters are enough:

```java
Person person = random(Person.class);
```
This snippet is readable as in plain english. When the user want to override any parameter, it is always possible to instantiate a `EnhancedRandom` using the builder and call `nextObject` method.

@punkratz312 What do you think of this addition?

**Comment by PascalSchumacher** [comment]
:+1:

**Comment by punkratz312** [comment]
@benas looks very nice. I could replace all my usages of the builder, if the excludedFields could be passed as well.

Again thanks for the help & and nice tool

**Comment by benas** [comment]
Thank you for your feedback.

I'll add the `excludedFields` parameter to the static method as well.

**Comment by benas** [comment]
The API has [been updated](https://github.com/benas/random-beans/blob/master/random-beans/src/main/java/io/github/benas/randombeans/api/EnhancedRandom.java#L81) in the master branch.

This will be released in v3 soon.

If you have any issue, please don't hesitate to file a bug.

Kind regards
Mahmoud
